src/block-management/block-manager.js

```diff
@@ -44,6 +51,10 @@ export default class BlockManager extends React.Component<PropsType, StateType>
 		};
 	}
 
+	componentDidMount() {
+		this.serializeToHtml();
```

Comment:
This function returns string, so it should be assigned to a variable.

Reply:
This was an experiment and I forgot to remove it 😞 
Thanks for catching it!